lib/matplotlib/figure.py

```diff
@@ -273,6 +273,14 @@ class Figure(Artist):
     def __str__(self):
         return "Figure(%gx%g)" % tuple(self.bbox.size)
 
+    def __repr__(self):
+        return "<{clsname} size {h:g}x{w:g} with {naxes} ax{ax_plural}>".format(
+            clsname=self.__class__.__name__,
+            h=self.bbox.size[0], w=self.bbox.size[1],
+            naxes=len(self.axes),
+            ax_plural='is' if len(self.axes) == 1 else 'es'
```

Comment:
The singular of Axes is Axes (... for matplotlib, Axis is something completely different (sad but true)).
TBH I would just have the suptitle and be done with it.

Reply:
Yeah, I think just saying "with one axes" is probably fine and technically more accurate, would it be possible to change that @richardjgowers ?

Reply:
@dstansby ok I've changed this